plugins/web/opentelemetry-instrumentation-document-load/README.md

```diff
@@ -82,6 +82,24 @@ Because the browser does not send a trace context header for the initial page na
 </body>
 ```
 
+## Optional : Add custom attributes to document load span if needed
+
+If it is needed to add custom attributes to the document load span,and/or document fetch span and/or resource fetch spans, respective functions to do so needs to be provided
+as a config to the DocumentLoad Instrumentation as shown below.
+
+```js
+
+const addCustomAttributesToDoucmentLoadSpan = (span)=> {
+    span.setAttribute('<custom.attribute.key>','<custom-attribute-value>');
+}
+registerInstrumentations({
+    instrumentations: [
+        new DocumentLoadInstrumentation({applyCustomAttributesOnDocumentLoadSpan:addCustomAttributesToDoucmentLoadSpan,
+                                         applyCustomAttributesOnDocumentFetchSpan:addCustomAttributesToDoucmentLoadSpan,
+                                         applyCustomAttributesOnResourceFetchSpans:addCustomAttributesToDoucmentLoadSpan})]
```

Comment:
I find it a bit weird that the example fills a property named `applyCustomAttributesOnDocumentFetchSpan` with a value `addCustomAttributesToDoucmentLoadSpan`.

Maybe rename the example implementation function to `addCustomAttributesToSpan`?